src/jsifier.js

```diff
@@ -326,7 +326,7 @@ function JSify(functionsOnly) {
     }
 
     itemsDict.functionStub.push(item);
-    item.JS = addFromLibrary(item, 'top-level compiled C/C++ code');
+    item.JS = addFromLibrary(item, 'top-level compiled C/C++ code', true);
```

Comment:
Instead of adding a parameter, how about turning `'top-level compiled C/C++ code'` into a named constant, and comparing to that to see if we are the top level? `if (dependent == TOP_LEVEL)` or such

Reply:
Done